content/CORE/Storage/Pools/StorageEncryption.md

```diff
@@ -50,136 +49,273 @@ These algorithms provide authenticated encryption with block ciphers.
+
+![EncryptedDatasetLockOptions](/images/CORE/12.0/EncryptedDatasetLockOptions.png "Encrytped Dataset Locking Options")
 
 Use the **Force unmount** option only if you are certain no one is currently accessing the dataset.
-After locking a dataset, the unlock icon changes to a locked icon.
-While the dataset is locked, it is not available for use.
+Click the **Confirm** checkbox to check-mark it and click **LOCK**, and then click **LOCK**.
+A confirmation window diplays indicating the dataset is locked and the unlock icon changes to a locked icon.
 
-To unlock a dataset, click <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> and *Unlock*.
+{{< hint info >}}
+You *cannot* use locked datasets.
+{{< /hint >}}
+
+#### Unlocking a Dataset
+
+To unlock a dataset, click on the <i class="material-icons" aria-hidden="true" title="Options">more_vert</i> icon to display the **Dataset Actions** menu and then click on **Unlock**.
 
 ![StoragePoolsDatasetUnlockOptions](/images/CORE/12.0/StoragePoolsDatasetUnlockOptions.png "Dataset Unlock Options")
 
-Enter the passphrase and click **Submit**. To unlock child datasets, set the *Unlock Children* box. Child datasets that inherited the parent dataset's encryption settings unlock when the parent unlocks. Users can unlock child datasets with different passphrases as the parent simultaneously by entering their passphrases.
+Type the passphrase and click **SUBMIT**. 
+To unlock child datasets, set the **Unlock Children** checkbox. 
+Child datasets that inherited the parent dataset's encryption settings unlock when the parent unlocks. 
+Users can unlock child datasets with different passphrases as the parent simultaneously by entering their passphrases.
 
-Confirm unlocking the datasets and wait for a dialog to show the unlock is successful.
+Two confirmation windows display. 
+Click **CONTINUE** to confirm you want to unlock the datasets or **CANCEL** to exit and keep the datasets locked. 
+A second confirmation window displays confirming the datasets are unlocked. 
+Click **CONTINUE** to close the second confirmation window. 
 
 ![StoragePoolsDatasetUnlockSuccess](/images/CORE/12.0/StoragePoolsDatasetUnlockSuccess.png "Dataset Unlock Success")
 
-**Example:**
+TrueNAS displays the dataset with the unlocked icon.
+
+**Unlocking Mutliple Datasets Example:**
 
 ![StoragePoolsDatasetUnlockexample1](/images/CORE/12.0/EncrytionExample1.png "Encrypted locked Datasets")
 
-The parent dataset is media. It has three child datasets. The `documents` child dataset has "inherited" the parent encryption settings and its password. The other two child datasets (`audio` and `video`) have their own passphrases. When the parent dataset is locked, all child datasets lock too.
+The parent dataset is media. 
+It has three child datasets. 
+The `documents` child datasetinherits the parent encryption settings and its password. 
+The other two child datasets (`audio` and `video`) have their own passphrases. 
```

Comment:
Suggested syntax switch to embolden names of datasets called out from example image:
```suggestion
The parent dataset is **media**.
It has three child datasets.
The **documents** child dataset inherits the parent encryption settings and its password.
The other two child datasets (**audio** and **video**) have their own passphrases.
```